docs/tutorials/promtail.md

```diff
@@ -10,6 +10,8 @@ If you prefer the video tutorial, check out the tutorial below on the Aqua Open
 
 ## Prerequisites
 
+### CLI tools
```

Comment:
I think this section can be summarize something like:

To follow along this tutorial you will need:
1. working k8s with cli tools
2. tracee installed (link to installation doc)
3. graphana stuff... (link to graphing doc?)
4. ...

If you already have these installed, you can skip to ...
otherwise, keep reading as we will setup the required tools

Reply:
hmm I thought we decided in the call to move the Tracee part to the top; you think it should be at the end?